python/housinginsights/sources/cama.py

```diff
@@ -0,0 +1,208 @@
+from pprint import pprint
+import os
+import sys
+import requests
+from collections import OrderedDict
+import csv
+import datetime
+
+PYTHON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+sys.path.append(PYTHON_PATH)
+
+from housinginsights.sources.base import BaseApiConn
+from housinginsights.sources.models.mar import MarResult, FIELDS
+
+
+class MarApiConn(BaseApiConn):
+    """
+    API Interface to the Master Address Record (MAR) database.
+    Use public methods to retrieve data.
+    """
+
+    BASEURL = 'http://citizenatlas.dc.gov/newwebservices/locationverifier.asmx'
+
+    def __init__(self):
+        super().__init__(MarApiConn.BASEURL)
+
+    def get_data(self, square, lot, suffix):
+        """
+        Get information on a location based on a simple query string.
+        
+        :param square: SSL first part
+        :type  location: String.
+        :param lot: SSL second part
+        :type  location: String.
+        :param output_type: Output type specified by user.
+        :type  output_type: String.
+        :param output_file: Output file specified by user.
+        :type  output_file: String
+        :returns: Json output from the api.
+        :rtype: String
+        """
+
+        params = {
+            'f': 'json',
+            'Square': square,
+            'Lot': lot,
+            'Suffix': suffix
+        }
+
+        result = self.get('/findAddFromSSL2', params=params)
+        if result.status_code != 200:
+            err = "An error occurred during request: status {0}"
+            raise Exception(err.format(result.status_code))
+
+        mar_data = result.json()
+        if mar_data['returnDataset'] == {}:
+            mar_returns = {'Warning': 'No MAR data availble - property under construction - see AYB year'}
+        else:
+            entry = mar_data['returnDataset']['Table1'][0]
+            mar_returns = {'anc': entry['ANC'],
+                           'census_tract': entry['CENSUS_TRACT'],
+                           'neighborhood_cluster': entry['CLUSTER_'],
+                           'ward': entry['WARD'],
+                           'zip': entry['ZIPCODE']
+            }
+
+        return mar_returns
+
+class CamaApiConn(BaseApiConn):
+    """
+    API Interface to the Master Address Record (MAR) database.
```

Comment:
This docstring needs to edited, looks like it was copy/pasted. Would be good to be more descriptive - e.g. what is Cama, what kind of data do we retrieve from cama